scripts/migrateHistoricBids.ts

```diff
@@ -68,13 +68,13 @@ async function getAllFromCollection<T>(
   return result;
 }
 
-async function writeMessagesToOutputFile<T>(messages: TypedMessage<T>[][]) {
+async function writeMessagesToOutputFile<T>(messages: TypedMessage<any>[][]) {
```

Comment:
We can drop the `T` from the method signature if we won't be using it internally. Although this does beg the question of why we're giving up the type variable here